### DISCUSSION TITLE
Test version

### DISCUSSION THREAD

**Discussion by ImSammy6** [Maplespe/DWMBlurGlass#345] [Q&A] [answered]
Apparently the test version adds a glow effect to the buttons, but idk how to compile it. Is there a download for it anywhere?

**Comment by Undisputed00x** [comment, marked as answer]
https://github.com/Maplespe/DWMBlurGlass/discussions/282#discussioncomment-9245189

**Reply by ImSammy6** [reply]
tysm dude
